webgpu.h

```diff
@@ -1566,6 +1568,37 @@ typedef struct WGPUAdapterInfo {
     /*.deviceID=*/0 _wgpu_COMMA \
 })
 
+/**
+ * Enabled by the "subgroups" feature.
+ * Provides bounds on the sizes of subgroups supported by the adapter.
```

Comment:
Would be worth precising what base structure this can be chained to (is it `WGPUAdapterInfo`?).